src/Microsoft.Docs.LearnValidation/LearnValidationLogger.cs

```diff
@@ -8,13 +8,7 @@ namespace Microsoft.Docs.LearnValidation
 {
     public class LearnValidationLogger
     {
-        /// <summary>
-        /// Delegate to write to .errors.log file
-        /// </summary>
-        private readonly Action<LearnLogItem> _writeLog;
-        private readonly HashSet<string> _filesWithError;
-
-        private static readonly Dictionary<LearnErrorCode, string> s_errorMessageMapping = new()
```

Comment:
these lines seem no need to change

Reply:
Fixed, due to the false positive of visual studio.